SCons/Platform/Platform.xml

```diff
@@ -121,50 +121,123 @@ else:
+        that from the &Python; <systemitem>platform.machine</systemitem> method.
+</para>
+<para>
+        On the Windows platform, if the Microsoft Visual C++ compiler
+        is available, &t-link-msvc; tool setup is done using
+        &cv-HOST_ARCH; and &cv-link-TARGET_ARCH;. Changing the values at
+        any later time will not cause the tool to be reinitialized.
+        Valid values for this usage are the same as for &cv-link-TARGET_ARCH;.
+</para>
+<para>
+        On other platforms, this &consvar; is informational and does
+        not currently affect construction activities.
 </para>
     </summary>
 </cvar>
 
 <cvar name="TARGET_ARCH">
     <summary>
 <para>
-        The name of the target hardware architecture for the compiled objects
-        created by this Environment.
-        This defaults to the value of HOST_ARCH, and the user can override it.
-        Currently only set for Win32.
+        The name of the hardware architecture compiled objects
+        created by this &consenv; should target.
+        Can be set when creating a &consenv; by passing as a keyword
+        argument in the &f-link-Environment; call; should not be
+        changed otherwise.
+        If not specified in the &f-Environment; call, will be set to
+        the defaults for the particular platform, which is in turn
+        auto-detected unless the <parameter>platform</parameter>
+        argument to &f-Environment; is given.
+        Defaults to the same value as &cv-link-HOST_ARCH;.
 </para>
-    </summary>
+<para>
+        On the Windows platform, if the Microsoft Visual C++ compiler
+        is available, &t-link-msvc; tool setup is done using
+        &cv-link-HOST_ARCH; and &cv-TARGET_ARCH;. Changing the values at
+        any later time will not cause the tool to be reinitialized.
+        Compiled objects will be in the target architecture if
+        the compilation system supports generating for that target.
+        The latest compiler which can fulfill the requirement will
+        be selected, unless a different version is directed by the
+        value of the &cv-link-MSVC_VERSION; &consvar;.
+</para>
+<para>
+        On the Windows/msvc combination, valid target values are
+        <literal>x86</literal>,
+        <literal>arm</literal>,
+        <literal>i386</literal>
+        for 32-bit targets and
+        <literal>amd64</literal>,
+        <literal>arm64</literal>,
+        <literal>em64t</literal>,
+        <literal>x86_64</literal>
+        and <literal>ia64</literal> (Itanium)
+        for 64-bit targets.
+        Note that not all target architectures are
+        supported for all Visual Studio / MSVC versions.
+        Check the relevant Microsoft documentation.
+</para>
+<para>
+        For example, if you want to compile 64-bit binaries, you would set
+        <literal>TARGET_ARCH='x86_64'</literal> when creating the &consenv;.
+</para>
+<para>
+        On other platforms, this &consvar; is informational and does
```

Comment:
I'd remove this snippet as well. Lest we have to go through deprecation cycle if it changes.

Reply:
I'd rather not imply that setting it accomplishes something (like cross-compiling magically working).

Reply:
Maybe. `Currently this only has any effect for MSVC compilers.`. A little less open ended?